pkg/operator/ceph/cluster/mon/service.go

```diff
@@ -53,7 +42,10 @@ func (c *Cluster) createService(mon *monConfig) (string, error) {
 		return "", errors.Wrapf(err, "failed to set owner reference to mon service %q", svcDef.Name)
 	}
 
-	// If deploying Nautilus or newer we need a new port for the monitor service
+	// If the cluster doesn't require msgr2 we allow msgr1 as well
+	if !c.ClusterInfo.RequireMsgr2 {
```

Comment:
Any reason not to only enable msgr2 and forget about msgr1?

Reply:
I think we still have to worry about old clients. Even the CI was not happy with enabling only msgr2, but I'm still investigating that. If we could just go with msgr2 it would certainly simplify things.